### ISSUE TITLE
Midterm Review - Akash Nadan (an376)

### ISSUE THREAD

**Issue by akashnadan1** [liufan2312/ORIE_4741_Project#6]
Overall, your midterm report is extremely well written and formatted. It is really nice that you guys included a table of contents and clearly label all diagrams in the report. A lot of the analysis that you guys have already done looks really good and insightful. One of my suggestions would be that in section 1.3 Cleaning Data, you talk about increasing the number of columns of the data set from 28 columns to 220 columns, and I think it would interesting to see if there is another way to account for missing values instead of creating additional columns. I think that increasing your column size will significantly add to your computation time in certain algorithms.

Additionally, in section 2.1 Model Selection you talk about splitting the data into 40% training, 20% validation, and 20% test. I think that instead of creating a 20% set that is your validation set. I think it might be better to use K-fold cross validation or LOOCV on a 60% training set. Furthermore, I think it would be interesting to run K-nearest neighbors on the problem that your are working on, or multi-class SVM. I think that both of the latter methods would do a good job at classifying the wells in your problem. Finally, it would be interesting to see if your model could generalize to wells in other areas outside of Tanzania, if you have the time.
